src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -63,8 +63,7 @@ export class DotEditLayoutDesignerComponent implements OnInit {
             .subscribe();
 
         this.setupLayout();
-
-        if (!this.isLayout() && this.pageState.template.canEdit) {
+        if (!this.isLayout() && this.pageState.template.canEdit && this.editTemplate) {
```

Comment:
Move the this.editTemplate because if thats false we don't need to check anything else.